src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -333,4 +340,26 @@ describe('Visual refresh toolbar only', () => {
       await expect(page.isDisplayed('[data-testid="drawer-sticky-header"]')).resolves.toBe(true);
     })
   );
+
+  test(
+    'should show only expanded drawer and hide all other panels if expanded mode for a drawer is active',
+    setupTest(async page => {
+      await page.setWindowSize({ ...viewports.desktop, width: 1600 });
```

Comment:
There are several places with this 1600 width already

Is this time to create a new `viewports` constant, something like "width to fit 3 side-by-side drawers"